### ISSUE TITLE
Root of Wix.com added erroneously?

### ISSUE THREAD

**Issue by colindean** [StevenBlack/hosts#711]
It seems sensible to blacklist the tracking endpoints but blocking wix.com itself is preventing my users from accessing its website administration tool.

It was added here: https://github.com/StevenBlack/hosts/commit/3f6b6ce2c6d4cd45062d8bef4c9ffc2d17804f3f#diff-85cf4e6d42a71e693fd780c8b29accddR61599 in commit 3f6b6ce2c6d4cd45062d8bef4c9ffc2d17804f3f on Friday.

I'll put up a PR myself in a couple of hours if no one gets to it before I do.

**Comment by lightswitch05** [comment]
@colindean I can clean this up since it was added in my list lightswitch05@0b0fed36f5d9d22928d9d81802a7acc0a96d57eb

Looks like a bunch of subdomains were added. Can you confirm which ones in the list are for tracking? Then I'll remove the innocuous ones

**Comment by colindean** [comment]
Need to unblock:

```
www.wix.com            # www used for basic stuff
static.wixstatic.com   # image host
users.wix.com          # admin login page
wix.com                # bare domain used a lot it seems
editor.wix.com         # web editor
frog.wix.com           # appears to be some javascript? Maybe it can remain blocked?
```

I whitelisted all but `frog.wix.com` and their wix-hosted site and the editor appear to be accessible.

My list may not be comprehensive, though.

**Comment by colindean** [comment]
Unfortunately, I can't really tell which are used for tracking. `frog.wix.com` seems to be loaded on all pages several attempts but doesn't impede usage for my little tests and demos.

**Comment by lightswitch05** [comment]
fixed in lightswitch05@978d61fb6e3e60af11dcb9fccc41f3e9b6d6a101 please close the ticket. If you have more issues, lets continue in my issues tracker https://github.com/lightswitch05/hosts/issues

**Comment by colindean** [comment]
Closing per https://github.com/StevenBlack/hosts/issues/711#issuecomment-403619696

Thank you!
